server/src/api/donations/donations.service.ts

```diff
@@ -52,19 +52,20 @@ export const update = async (r: any): Promise<response> => {
     text: `
       UPDATE donations
       SET (
-        donorid,
+        donationid,
```

Comment:
This is the pkey which is being searched on, I think you may have wanted this to be `contactid_fk`